modules/modules.nix

```diff
@@ -92,7 +92,7 @@ let
     (loadModule ./services/flameshot.nix { })
     (loadModule ./services/gnome-keyring.nix { })
     (loadModule ./services/gpg-agent.nix { })
-    (loadModule ./services/imapnotify.nix { })
+    (loadModule ./services/imapnotify.nix { condition = hostPlatform.isLinux; })
```

Comment:
Why is this condition here? I don't see why it wouldn't work on macOS for example, it's just a node script (I must admit, I haven't dug deep in the source code to see if there are any incompatibilities).

Reply:
It's because the module uses systemd, which is only available for the Linux kernel.

Reply:
Ah, I suppose the surrounding lines ought to have that too then

Reply:
Yes, all modules using systemd should have it but unfortunately cannot because of backwards compatibility with people who reference them in their configurations on Darwin.